### PR TITLE
Added first version of Airbrake Simulator

### PR DESCRIPTION
**Pull request by mstoelzle**
This pull request does not provide any new functionality. It serves as a preparation of the implementation of a full airbrake simulation.

**Changes:**
1. The property `airbrake_extension` of the `rocket_state` is replaced with an `airbrake_state` array field
2. Plotting is added of the airbrake station and the airbrake position tracking error

### CODE REVIEW

File: src/eulersimcon_backend/components/plotting/flight_plotter.py [hunk 1]
```diff
@@ -312,11 +309,48 @@ def plot_control(self, flight_data: Dict):
...
                 plt.grid(True)
 
-            path_control_figure1 = os.path.join(self.path_plots, str(self.flight_model.id) + "_control_fig1.pdf")
-            plt.savefig(path_control_figure1)
+            fig.tight_layout()
+            plt.draw()
+
+            path_control_fig1 = os.path.join(self.path_plots, str(self.flight_model.id) + "_control_fig1.pdf")
+            plt.savefig(path_control_fig1, dpi=fig.dpi)
+            plt.show()
```
Comment (TunVunDerBunn) [inline src/eulersimcon_backend/components/plotting/flight_plotter.py:317]: can we remove in line 317 the plt.show() else programming will always be stuck here I think

Comment (TunVunDerBunn) [reply src/eulersimcon_backend/components/plotting/flight_plotter.py:317]: as there is already a plt.show() in line 352 all plots will be shown

File: src/eulersimcon_backend/components/simulation/airbrake_simulator.py [hunk 2]
```diff
@@ -11,10 +11,14 @@ def __init__(self):
     def run(self, rocket_state: RocketState, control_step: ControlStep = None) -> np.array:
         # airbrake_state: [airbrake position, airbrake velocity]
         # if control_step is None, the state dot should still be calculate but without any change in behaviour
-        if control_step is not None:
-            control_input = control_step.control_input[0]
-        airbrake_state = rocket_state.airbrake_state
 
-        airbrake_state_dot = np.array([airbrake_state[1], 0])
```
Comment (TunVunDerBunn) [inline src/eulersimcon_backend/components/simulation/airbrake_simulator.py]: why is the acceleration always set to zero? Or should that be changed in the future

Comment (mstoelzle) [reply src/eulersimcon_backend/components/simulation/airbrake_simulator.py]: This is just the current default until @christophdomenig98 implements the derivative equations

File: src/eulersimcon_backend/components/plotting/flight_plotter.py (hunk 1 above)
Comment (mstoelzle) [reply src/eulersimcon_backend/components/plotting/flight_plotter.py:317]: It does work. I just tested it and I implemented it in the same way for the state estimation. It works even better when you use PyCharm to run any commands. Then the plots will be directly shown in PyCharm

Comment (TunVunDerBunn) [reply src/eulersimcon_backend/components/plotting/flight_plotter.py:317]: okay perfect :)

**Pull request merged**